File: 프로그래머스/python/LV2/올바른 괄호.py
def solution(s):
    answer = True
    stack = []
    for i in s:
        if i == "(":
            stack.append(i)
        else:
            if stack == []:
                return False
            else:
                stack.pop()
    if len(stack) != 0:
        return False

    return True

File: 프로그래머스/javascript/LV2/올바른 괄호.js
function solution(s) {
    const stack = []

    for (let word of s) {
        if (word === "(") {
            stack.push(word);
        }
        else {
            if (stack.length === 0) {
                return false;
            }
            stack.pop();
        }
    }

    return stack.length === 0;
}